packages/office-ui-fabric-react/src/components/Button/CommandBarButton/CommandBarButton.styles.ts

```diff
@@ -209,6 +225,9 @@ export const getStyles = memoizeFunction(
 
       menuIcon: {
         color: p.neutralSecondary,
+        [HighContrastSelector]: {
+          color: 'GrayText',
```

Comment:
Looks like it's missing to set `border: none` here:
![image](https://user-images.githubusercontent.com/1207059/88951003-46797800-d24a-11ea-8cb7-48db5d926fae.png)

Reply:
This does not affect the border styling for the icon, SplitButtonMenuButtonDisabled does, updated to remove border